test/e2e/velero_utils.go

```diff
@@ -277,38 +271,6 @@ func VeleroInstall(ctx context.Context, veleroImage string, veleroNamespace stri
 	return nil
 }
 
-func VeleroUninstall(ctx context.Context, client *kubernetes.Clientset, extensionsClient *apiextensionsclient.Clientset,
```

Comment:
Let's change VeleroUninstall to call uninstall.Uninstall for the moment and later we can have it run through the CLI.

Reply:
@dsu-igeek , not sure if this is the right place to ask this but I can take a look at using cli in Install and Uninstall in the test suite since I have some context on what needs to be done, might as well make that change as well. Do we have any ticket for this or should I create a new one?